python/cudf_polars/cudf_polars/engine/options.py

```
@@ -338,8 +345,10 @@ class StreamingOptions:
     kvikio_statistics: bool | Unspecified = _opt(
         "executor", "CUDF_POLARS__EXECUTOR__KVIKIO_STATISTICS", parse_boolean
     )
-    max_concurrent_io_tasks: int | Unspecified = _opt(
-        "executor", "CUDF_POLARS__EXECUTOR__MAX_CONCURRENT_IO_TASKS", int
+    max_concurrent_io_tasks: int | dict[str, int] | None | Unspecified = _opt(
```

Ruff (0.16.2)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)